tools/dataset_fixtures.py

```diff
@@ -146,7 +146,7 @@ def get_start_end_date(date_string):
             'model': 'datasets.datafile',
             'pk': r['id'],
             'fields': {
-                'title': r.get('description', '')[0:64],
+                'name': r.get('description', '')[0:64],
```

Comment:
Yes. Originally the field was called title. And so the fixtures had title as the key.  We changed the model from title to name, but then forgot to update the fixtures.  So when you try and loaddata it was trying to fill the title field which no longer exists.